decisions/why-material-design/index.qmd

```diff
@@ -0,0 +1,44 @@
+---
+title: "Why Material Design"
+description: "We want to Seedcase to look good with a minimal effort."
+author: "Philip Christiansen"
+date: last-modified
+categories:
+---
+
+
+## Context and Problem Statement
+
+Seedcase should look good with a minimal effort. Therefore, we want to leverage existing CSS or UI frameworks rather than starting from scratch. 
```

Comment:
```suggestion
Seedcase should look good with minimal effort. Therefore, we aim to utilise existing CSS or UI frameworks instead of building from scratch.
```